build/android/bytecode/java/org/brave/bytecode/BraveDefaultBrowserPromoUtilsClassAdapter.java

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.brave.bytecode;
+
+import org.objectweb.asm.ClassVisitor;
+
+public class BraveDefaultBrowserPromoUtilsClassAdapter extends BraveClassVisitor {
+    static String sDefaultBrowserPromoUtilsClassName = "org/chromium/chrome/browser/ui/default_browser_promo/DefaultBrowserPromoUtils";
+
+    static String sBraveDefaultBrowserPromoUtilsClassName = "org/chromium/chrome/browser/ui/default_browser_promo/BraveDefaultBrowserPromoUtils";
+
+    public BraveDefaultBrowserPromoUtilsClassAdapter(ClassVisitor visitor) {
+        super(visitor);
+
+        changeMethodOwner(sDefaultBrowserPromoUtilsClassName, "prepareLaunchPromoIfNeeded", sBraveDefaultBrowserPromoUtilsClassName);
```

Comment:
Please a test to `testMethodsExist` in `BytecodeTest.java` to make sure `prepareLaunchPromoIfNeeded` is still in `DefaultBrowserPromoUtils` in further Chromium versions.

Reply:
done. can you please check if it is proper?